.github/actions/base/action.yml

```diff
@@ -42,36 +33,9 @@ runs:
         ghc --info
 
     - name: Install LLVM/Clang
-      uses: KyleMayes/install-llvm-action@v2.0.8
+      uses: ./.github/actions/setup-llvm
       with:
         version: ${{ inputs.llvm }}
-        directory: ${{ runner.temp }}/llvm
-        env: false
-
-    - name: macOS quirks - Configure environment variables
-      shell: bash
-      if: ${{ startsWith(inputs.runner, 'macos') }}
-      run: |
-        echo "LLVM_CONFIG=$LLVM_PATH/bin/llvm-config" >> "$GITHUB_ENV"
-        echo "LIBCLANG_PATH=$LLVM_PATH/lib/" >> "$GITHUB_ENV"
-        # https://stackoverflow.com/questions/63342521/clang-on-macos-having-problems-with-its-includes
-        export SDKROOT=$(xcrun --show-sdk-path --sdk macosx)
-        echo $SDKROOT
-        echo "SDKROOT=$SDKROOT" >> "$GITHUB_ENV"
-
-    - name: Debug (LLVM/Clang)
```

Comment:
Is there a reason why we removed the debug output? It is quite useful to know which version of the N installed versions is actually used :-P